ingest.py

```diff
@@ -1,15 +1,16 @@
 #!/usr/bin/env python3
-
-# ingest.py
 """
-Source client for Voctomix.
+ingest.py: source client for Voctomix.
+
+Ryan Verner <ryan@nextdayvideo.com.au>
+Carl Karsten <carl@nextdayvideo.com>
```

Comment:
TBH i don't care what order this in, feel free to juggle, just good to have this here IMO esp if it'll be part of voctomix's example scripts